rust/libnewsboat/src/configpaths.rs

```diff
@@ -9,6 +9,8 @@ use std::os::unix::fs::DirBuilderExt;
 use std::path::{Path, PathBuf};
 use xdg;
 
+use utils;
```

Comment:
We don't split imports into sections at the moment, so please put that import before `use xdg`.